docker/Dockerfile.olbase

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:xenial
+FROM python:3.9.4
```

Comment:
Py3.9.6 is current  https://www.python.org/downloads

Now avail... Py3.10 release candidate 1... https://pythoninsider.blogspot.com/2021/08/python-3100rc1-is-available.html

Reply:
One step at a time :D